lib/ansible/modules/cloud/amazon/ec2_ami.py

```diff
@@ -626,13 +626,16 @@ def get_image_by_id(module, connection, image_id):
         module.fail_json_aws(e, msg="Error retrieving image by image_id")
 
 
-def rename_item_if_exists(dict_object, attribute, new_attribute, child_node=None):
+def rename_item_if_exists(dict_object, attribute, new_attribute, child_node=None, type=None):
     new_item = dict_object.get(attribute)
     if new_item is not None:
+        value = dict_object.get(attribute)
+        if type is not None:
```

Comment:
I don't really like overriding the `type` builtin - can we use `attribute_type` as  parameter or similar?